data/json/construction.json

```diff
@@ -850,6 +850,38 @@
     "pre_terrain": "t_pit_shallow",
     "post_terrain": "t_woodchips"
   },
+  {
+    "type": "construction",
+    "description": "Make Gravel Floor",
+    "//": "Covers the floor with pebbles. Used for constructing railroads.",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 1 ] ],
+    "time": "40 m",
+    "components": [ [ [ "pebble", 100 ] ] ],
+    "post_terrain": "t_railroad_rubble"
+  },
+  {
+    "type": "construction",
+    "description": "Build Straight Small Railroad Track",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 4 ] ],
+    "time": "30 m",
+    "qualities": [ [ { "id": "HAMMER", "level": 1 } ], [ { "id": "WELD", "level": 1 } ] ],
```

Comment:
HAMMER: 1 is pretty much a decently shaped rock, I think this calls for hammer: 2.
OTOH I'm not clear what the welding is for, are you butting the tracks and welding them instead of using duckplates?

Reply:
I was thinking of welding at first, but using duckplates is more in line with the process shown in the video. added 2 small metal sheets. also hammer 2